lib/safer_rails_console/patches/sandbox/redis_readonly.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module SaferRailsConsole
+  module Patches
+    module Sandbox
+      module RedisReadonly
+        # rubocop:disable Style/WordArray
+        WRITE_COMMANDS = %w{
+          bitop restore pfdebug lpushx incr move getex decrby renamenx flushdb setex
+          setnx linsert rpush bzpopmax hset del copy xsetid georadiusbymember setrange blmove set rpop
+          lset xtrim zrangestore psetex xclaim append georadius incrbyfloat bitfield expire sort hsetnx
+          sadd zincrby lpop spop sunionstore getdel lrem zunionstore sdiffstore zremrangebyscore ltrim
+          bzpopmin xack pfadd unlink swapdb sinterstore zrem xgroup hdel zdiffstore xautoclaim xdel hmset
+          zpopmax zremrangebyrank setbit pexpireat mset hincrbyfloat incrby blpop getset expireat xreadgroup
+          hincrby migrate lmove pexpire flushall smove msetnx decr persist rpushx pfmerge xadd zremrangebylex
+          restore-asking geoadd rpoplpush zadd lpush srem brpoplpush zpopmin brpop geosearchstore zinterstore rename
+        }.freeze
+        # rubocop:enable Style/WordArray
+
+        def self.raise_exception_on_write_command(command)
```

Comment:
How should this handle eval, https://redis.io/commands/eval/, and evalsha, https://redis.io/commands/evalsha/?